backend/api/Services/MissionRunService.cs

```diff
@@ -58,13 +58,28 @@ public class MissionRunService(
         ILogger<MissionRunService> logger,
         IAccessRoleService accessRoleService) : IMissionRunService
     {
+
+        private static readonly SemaphoreSlim missionRunCountSemaphore = new(1);
```

Comment:
Does this work in a transient service? And is it the reason you made the service transient?

Reply:
As discussed, seems to be working while testing